docs/serving/services/storage.md

```diff
@@ -1,12 +1,35 @@
 # Volume Support for Knative services
 
-By default Serving supports the mounting the [volume types](https://kubernetes.io/docs/concepts/storage/volumes): `emptyDir`, `secret`, `configMap` and `projected`. [PersistentVolumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) are supported but require a [feature flag](../configuration/feature-flags.md) to be enabled.
+You can provide data storage for Knative Services by configuring different types of volumes.
+Serving supports mounting the [volume types](https://kubernetes.io/docs/concepts/storage/volumes): `emptyDir`, `secret`, `configMap` and `projected`.
+[PersistentVolumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) are supported but require a [feature flag](../configuration/feature-flags.md) to be enabled.
 
 !!! warning
     Mounting large volumes may add considerable overhead to the application's start up time.
 
 
-Here is an example of using a persistent volume claim with a Knative Service:
+Bellow there is an example of using a persistent volume claim with a Knative Service.
+
+## Prerequisites
+
+Before you can configure PVCs for a Service, this feature must be enabled in the `KnativeServing` ConfigMap:
```

Comment:
Do we also want to document without the operator? For the yaml install a modification of the config-map?

Reply:
Actually we can have a note about the operator but this is the first feature described separately, most are handled in the feature-flags page and there is no mentioning of the operator there. It is only about the config map.

Reply:
> in the `KnativeServing` ConfigMap:

But then this is not correct, right? It is either the Custom Resource `KnativeServing` or the `config-features` ConfigMap.

Reply:
Listing one option does not exclude the other. For completeness sake we could add both options.
So when you deploy Serving with the Operator your API is the Serving CR so config should be changed there.
Operator will propagate the change to the configmap as usual.
When you dont deploy Serving with the Operator configmap is your place where to changed stuff.